assets/stylesheets/responsive/custom.scss

```diff
@@ -575,6 +575,61 @@ dt {
     }
 }
 
+/* How it works section */
+.steps__list {
+  li {
+    @include respond-min( $main_menu-mobile_menu_cutoff ){
+      width: 33.3333%;
+      margin-left: 0 !important;
+    }
+  }
+}
+
+.steps__step-box {
```

Comment:
This might look a bit better if we apply the same shadow as on the correspondence boxes?

![screen shot 2015-08-04 at 13 00 07](https://cloud.githubusercontent.com/assets/282788/9060132/e1ea6554-3aa8-11e5-8b32-463d36e46caf.png)

I think the white directly on the cream is a little hard on the eyes – open to options of course!